src/config.rs

```diff
@@ -87,7 +87,8 @@ impl Density {
     pub fn to_list_tactic(self) -> ListTactic {
         match self {
             Density::Compressed => ListTactic::Mixed,
-            Density::Tall | Density::CompressedIfEmpty => ListTactic::HorizontalVertical,
```

Comment:
The idea is to put simple patterns on the same line (or mixed), so something `Foo | Bar | Baz` doesn't end up like

```
Foo |
Bar |
Baz
```